Models/SwissBitcoinPayInvoiceModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+namespace SmartStore.SwissBitcoinPay.Models
+{
+
+    public struct SwissBitcoinPayInvoiceModel
+    {
+        public string title;
+        public string description;
+        public decimal amount;
+        public string unit;
+        //public bool onChain;

```

Comment:
You should add a boolean setting "Accept Onchain payments"

Reply:
How should this setting be managed in the plugin ?

Reply:
OK, done.
I've added a flag in the merchant configuration to allow to accept onchain payments